docs/_getting-started/index.md

```diff
@@ -38,39 +38,65 @@ For this tutorial, you will use a `light` sync:
 - Basic Blockchain Knowledge
 
 ## Step 1: Open Terminal
-You will need your system terminal to run the commands for this tutorial.
+
+You will need your system terminal to run the commands for this tutorial,
+
+Use the command below to create an account 
 
 ![Create new account command](../../static/images/open_terminal.png)
 ## Step 2: Create accounts
 
 Use the command below to create an account 
 > **Note:** you will need to create two accounts for this guide
 
-```shell
+```javscript
 clef newaccount --keystore geth-tutorial/keystore
 ```
 
 It will give you the result below:
 
-![Create new account command](../../static/images/create_account.png)
+```
+WARNING! 
+Clef is an account management tool. It may, like any software, contain bugs. 
+
+Please take care to 
+— backup your keystore files,
+ — verify that the keystore(s) can be opened with your password. 
+
+Clef is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR 
+PURPOSE. See the GNU General Public License for more details. 
+
+Enter 'ok' to proceed: 
+> ok 
+```
 
 Enter “ok” and hit the enter key. Next, the system will request the below action.
+
 **Please enter a password for the new account to be created (attempt 0 of 3)**
+
 Enter your desired password and hit the enter key to get the result below:
 
-![Create new account command](../../static/images/createaccount2.png)
+```
+INFO [02-07118:19:57.914] Your new key was generated                                       address=0xca57F3b40842FCce3c3713881848ca5260ce72EC
+WAN. [02-07118:19:57.915] Please backup your key file, b40642fcce3c37b8d18adbca5260ca72ec  path=Users/wIsdommokochanocuments/LitHub/GethExample/getb-tutortalikeystore/UTC-2022-02-07717-19-58.5175380002—Ca57f3 
+WAN! [02-07118:19:57.915] Please remember your password! 
```

Comment:
There are multiple strange typos in this text now. I think these happened because you OCRed the images. It would be better to regenerate these texts instead of trying to get them from the image.